zerver/views/custom_profile_fields.py

```diff
@@ -56,21 +64,35 @@ def delete_realm_custom_profile_field(request: HttpRequest, user_profile: UserPr
                                          field=field)
     return json_success()
 
+def validate_field_length(name: str, value: str, max_length: int) -> None:
+    if len(value) > max_length:
+        msg = _("Length of '{field_name}' should be be less than or equal to {max_length}.")
+        raise ValueError(msg.format(field_name=name, max_length=max_length))
+
 @require_realm_admin
 @has_request_variables
 def update_realm_custom_profile_field(request: HttpRequest, user_profile: UserProfile,
-                                      field_id: int, name: Text=REQ()) -> HttpResponse:
+                                      field_id: int, name: Text=REQ(),
+                                      hint: Text=REQ(default='')
+                                      ) -> HttpResponse:
     if not name.strip():
         return json_error(_("Name cannot be blank."))
 
+    try:
+        max_length = CustomProfileField.HINT_MAX_LENGTH
+        validate_field_length('hint', hint, max_length)
```

Comment:
We don't need to define a new validator; just do something with `check_capped_string`, e.g. `hint: Text = REQ(validator=check_capped_string(...)`.  Might need to refactor a bit to turn `check_capped_string` into a decorator, so that `check_capped_string(CustomProfileField.HINT_MAX_LENGTH)` returns a function that checks the length is below that.

Reply:
`REQ(default='', validator=check_capped_string)` doesn't work because `REQ` complains that the data is not a valid json.

Reply:
I think I missed this. Doing it now.

Reply:
Done.